src/app/utils/bibDetailsUtils.jsx

```diff
@@ -212,6 +212,73 @@ const constructSubjectHeadingsArray = (url = '') => {
     });
 };
 
+/**
+ * isRtl(string)
+ * Returns true (or false) depending on whether the string needs to be read right to left
+ * @param {string} string
+ * @return  {boolean}
+ */
+const isRtl  = (string) => {
+  if (typeof string !== 'string') { return null }
+  return string.substring(0, 1) === '\u200F'
+}
+
+/**
+ * stringDirection(string)
+ * 'rtl' if the string needs to be read right to left, otherwise null
```

Comment:
I was thinking that "ltr" could be used as the default but I'm not sure if that's the correct behavior. It might not hurt for the app's purpose since we expect user's default to be "ltr".

Reply:
And since browsers already use "dir='lrt'", then adding this would have no effect.

Reply:
What I like about using `null` is that the resulting html will just not include a `dir` attribute, which seems 1. Closer to the existing functionality and 2. more semantically correct than enforcing `dir=ltr`.

Reply:
Yeah, that makes sense.